src/components/PeopleTable/PeopleTableRow.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Tr,
+  Td,
+  VStack,
+  HStack,
+  Flex,
+  Avatar,
+  Badge,
+  Text,
+  IconButton,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+} from '@chakra-ui/react';
+import { BsThreeDotsVertical } from 'react-icons/bs';
+import styles from './PeopleTable.module.css';
+
+const NameColumn = ({ data }) => {
+  return (
+    <Flex w="284px" h="72px" gap="12px" alignItems="center">
+      <Avatar size="md" name={data.name} src="something" />
+      <VStack alignItems="flex-start">
+        <Text>{data.name}</Text>
+        <Text color="#718096">{data.email}</Text>
+        {data.training ? (
+          <Badge
+            className="training-badge"
+            variant="capitalize"
+            bgColor="ochOrange"
+            fontSize="12px"
+            alignSelf="stretch"
+            borderRadius="6px"
+            padding="2px 8px"
+          >
+            In-Training
+          </Badge>
+        ) : null}
+      </VStack>
+    </Flex>
+  );
+};
+
+const SegmentColumn = ({ data }) => {
+  return (
+    <HStack w="100%" justifyContent="space-between">
+      <VStack align="normal">
+        {data?.assignedSegments.map((segment, i) => (
+          // eslint-disable-next-line react/no-array-index-key
+          <HStack key={i} alignItems="baseline">
+            <Text>{segment.name}</Text>
+            <Text color="#718096">{segment.description}</Text>
+          </HStack>
+        ))}
+      </VStack>
+      <VStack justify="right">
+        <RowButton data={data} />
+      </VStack>
+    </HStack>
+  );
+};
+
+const editAccountInfo = ({ data }) => {
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'data' is defined but never used.